Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
You can use a slug as primary key for example to have user-friendly url or you can just don't want the user to know the object id (to not know the object before/after for example).

IMO, this feature improves user comfort.